src/modules/tip/libkvitip_ru.kvc

```diff
@@ -25,5 +25,5 @@ TranslatorHint=This-file-MUST-be-encoded-in-UTF8
 20=KVIrc имеет встроенный клиент HTTP, и вы можете загрузить файлы из интернета используя команду <b>/http.get</b>.
 21=Вам не нравится окно с деревом панели задач? Вы можете выбрать традиционную панель задач (такую же как в mIRC) в секции настроек панели задач.
 22=Вы можете открыть соединения для многих серверов в одном окне KVIrc, просто выберите "Новый IRC контекст" в меню KVIrc.
-23=KVIrc имеет встроенный гибкий механизм идентификации на NickServ. Если вы заинтересованы в его использовании, но не можете настроить самостоятельно, советуем прочитать инструкции на сайте http://www.kvirc.ru в разделе статьи.
-24=Это последний совет. Вы можете добавлять свои собственные советы - наберите <b>/help tip.open</b>. Вы можете отослать свои советы (на любом языке) по адресу <b>alexey@kvirc.ru</b>.
+23=KVIrc имеет встроенный гибкий механизм идентификации на NickServ. Если вы заинтересованы в его использовании, но не можете настроить самостоятельно, советуем прочитать инструкции на сайте http://www.kvirc.net в разделе статьи.
```

Comment:
This sentence doesn't make sense anymore, because kvirc.net doesn't have the Articles page which explains how to setup Nickserv. I suggest replacing it with this:

23=KVIrc имеет встроенный гибкий механизм идентификации на NickServ. Настроить его можно в списке серверов: в свойстве IRC-сервера на вкладке Дополнительно в разделе SASL, а также в свойстве IRC-сети на вкладке NickServ.

Reply:
Done, thank you